skiko/buildSrc/src/main/kotlin/utils.kt

```diff
@@ -43,3 +43,17 @@ fun Task.projectDirs(vararg relativePaths: String): List<Directory> {
     val projectDir = project.layout.projectDirectory
     return relativePaths.map { path -> projectDir.dir(path) }
 }
+
+fun listOfFrameworks(vararg frameworks: String): List<String> =
+    frameworks.flatMap { listOf("-framework", it) }
+
+fun mutableListOfLinkerOptions(options: Collection<String>): MutableList<String> =
```

Comment:
Can we also use `flatMap` here to simplify the code? (otherwise a reader should understand the difference between these two)